Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance-WidgetExtension.xcscheme

```diff
@@ -90,7 +90,6 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES"
-      askForAppToLaunch = "Yes"
```

Comment:
you may need to manually add this back, it shouldn't be removed

Reply:
easiest way is to open in VSCode and add this back manually
Also you can just push new commits, the PR will update automatically